doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -65,59 +96,61 @@ Peripheral samples
+     * :kconfig:option:`SB_CONFIG_MATTER_OTA` Kconfig option has been added to enable or disable generating Matter OTA package during the building process.
+     * :kconfig:option:`CONFIG_CHIP_OTA_IMAGE_FILE_NAME` Kconfig option is deprecated and you need to use the :kconfig:option:`SB_CONFIG_MATTER_OTA_IMAGE_FILE_NAME` Kconfig option instead to define Matter OTA output filename.
 
-  .. note::
+   .. note::
 
-    If you want to build a sample without using sysbuild, you need to use the old Kconfig options.
+      If you want to build a sample without using sysbuild, you need to use the old Kconfig options.
 
 Libraries
 =========
 
 This section describes the changes related to libraries.
 
-MQTT helper
------------
+MQTT helper library
+-------------------
 
 .. toggle::
 
-   * For applications using the :ref:`lib_mqtt_helper` library:
+   For applications using the :ref:`lib_mqtt_helper` library:
 
-     * The ``CONFIG_MQTT_HELPER_CERTIFICATES_FILE`` Kconfig option is now replaced by :kconfig:option:`CONFIG_MQTT_HELPER_CERTIFICATES_FOLDER`.
+     * The ``CONFIG_MQTT_HELPER_CERTIFICATES_FILE`` is now replaced by :kconfig:option:`CONFIG_MQTT_HELPER_CERTIFICATES_FOLDER`.
        The new option is a folder path where the certificates are stored.
        The folder path must be relative to the root of the project.
 
        If you are using the :ref:`lib_mqtt_helper` library, you must update the Kconfig option to use the new option.
 
      * When using the :kconfig:option:`CONFIG_MQTT_HELPER_PROVISION_CERTIFICATES` Kconfig option, the certificate files must be in standard PEM format.
-       This means that the PEM files must not be converted to string format anymore.
+       This means that the PEM files need not be converted to string format anymore.
 
 FEM abstraction layer
 ---------------------
 
 .. toggle::
 
-   * For applications using :ref:`fem_al_lib`:
+   For applications using :ref:`fem_al_lib`:
 
      * The function :c:func:`fem_tx_power_control_set` replaces the function :c:func:`fem_tx_gain_set`.
-       The function :c:func:`fem_default_tx_output_power_get` replaces the function :c:func:`fem_default_tx_gain_get`.
+     * The function :c:func:`fem_default_tx_output_power_get` replaces the function :c:func:`fem_default_tx_gain_get`.
 
 Modem library
 -------------
 
 .. toggle::
 
+   For applications using :ref:`nrf_modem_lib_readme`:
 
-   * For applications using :ref:`nrf_modem_lib_readme`:
-     The option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ZEPHYR` is now deprecated.
-     Use the option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART` instead.
+     * The option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ZEPHYR` is now deprecated.
+       Use the option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART` instead.
```

Comment:
This can be a bit misleading to customers as the `CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART` is not enough to use the UART trace backend. It should be enabled with the `nrf91-modem-trace-uart` snippet, or by adding similar configuration in application overlays.